example/crisis-overview.config.json

```diff
@@ -5,6 +5,9 @@
     "src": "img/overview-map.jpg",
     "alt": "Syria/Iraq"
   },
+  "environment": {
+
```

Comment:
This needs the environment.content such as "http://crisis.rwdev.org/syria/" in order to assemble the full URL to the map image.